Magento_SalesRule/web/template/cart/totals/discount.html

```diff
@@ -1,19 +1,12 @@
 <!-- ko if: isDisplayed() -->
     <li class="order-summary__list-item">
-        <span class="order-summary__label">
-            <span
-                class="title"
-                data-bind="text: getTitle()"
-            />
-            <span
-                class="discount coupon"
-                data-bind="text: getCouponLabel()"
-            />
-        </span>
```

Comment:
why this was removed and replaced by hardcoded value `$t('Discount')`

Reply:
because `getTitle` returns 'Discount' plus all the discount names in parenthesis (screenshot). I was supposed to remove discount names 
![image](https://user-images.githubusercontent.com/49198312/87442244-58c1a800-c5f4-11ea-8822-172987a5566e.png)